Godeps/_workspace/src/github.com/contiv/contivmodel/client/contivModel.js

```diff
@@ -199,7 +199,7 @@ var EndpointGroupSummaryView = React.createClass({
 				<thead>
 					<tr>
 					
-					     
+					      
 					</tr>
```

Comment:
Yeah you can ignore the godep changes. It wont be there in final commit after I update godeps